### ISSUE TITLE
[Quest][Northrend] The lost Mistwhisper Treasure

### ISSUE THREAD

**Issue by Saboyna** [dalaranwow/dalaran-wow#3917]
http://wotlk.openwow.com/quest=12575

**Current behaviour**: Quest Item (Mistwhisper Treasure) bugged, and even tho if you manage to get quest item + kill the npc you cant turn in the quest.

**Expected behaviour**: Be able to turn in the quest. 

This quest holds up for going forth with getting rep for Oracle/Frenzyheart would be nice if fixed as soon as possible.

![wowscrnshot_120216_071007](https://cloud.githubusercontent.com/assets/22807074/20824642/b010400a-b85e-11e6-9562-134517fbd311.jpg)

**Comment by Rushor** [comment]
always include ids pls http://wotlk.openwow.com/quest=12575

**Comment by Rushor** [comment]
https://github.com/TrinityCore/TrinityCore/issues/15620

**Comment by Rushor** [comment]
DB/Quest:  The lost Mistwhisper Treasure - Always reward questcredit from Warlord Tartek

* special questevent from quest_template_addon is no longer needed
* moreover the credit will be rewarded to the whole group now
* also added some adjustments to SMART_ACTION_CALL_GROUPEVENTHAPPENS
* closes https://github.com/dalaranwow/dalaran-wow/issues/3917
* commit: 1fcdbcc265ac5cc078a009004e0eee51afb3e080

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by RevvingRose** [comment]
The quest "The Lost Mistwhisper Treasure" still does not work correctly. I have killed Tartek and I have the quest item, but I am not able to turn in the quest. #3917
![wowscrnshot_061917_003210](https://user-images.githubusercontent.com/29561376/27315702-b93e938c-5540-11e7-9f86-4c65e268414b.jpg)

**Comment by Rushor** [comment]
https://i.imgur.com/Hj4Vtfy.jpg https://i.imgur.com/CFABYs7.jpg

**Comment by Lothloryen** [comment]
Working great @Rushor , will be on live realm with the next maintenance.

**Comment by UncleCrust** [comment]
The quest "The Lost Mistwhisper Treasure" still does not work correctly. I have killed Tartek and I have the quest item, but I am not able to turn in the quest. #3917
